README.md

```diff
@@ -692,6 +692,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
   * [MapBBCodeShare](https://github.com/MapBBCode/share.mapbbcode.org) - Tool for sharing custom OSM maps. Support for annotated markers, polygons, lines, multi-format import/export, multiple layers, shortlinks. ([Demo](http://share.mapbbcode.org/)) `WTFPL/Other` `PHP`
   * [OpenGTS](http://www.opengts.org/) - Entry-level fleet tracking system. Supports variety of tracking devices and protocols. Comes with rich web-interface and reporting features. ([Demo](http://track.opengts.org/track/Track), [Source Code](https://sourceforge.net/projects/opengts/files/server-base/)) `Apache` `Java`
   * [Openstreetmap](http://www.openstreetmap.org/) - OpenStreetMap is a map of the world, created by people like you and free to use under an open license. ([Source Code](https://github.com/openstreetmap/openstreetmap-website)) `GPLv2` `Ruby`
+  * [OwnTracks](http://owntracks.org/) `⚠` - Location tracking for Android and iOS, embedded devices, and home automation. ([Source Code](https://github.com/owntracks/recorder)) `GPLv2` `C`/`Lua`
```

Comment:
Sure, I guess this makes more sense given the list's focus. I'll add commit to the branch, feel free to squash it / rebase it.